src/allmydata/scripts/common.py

```diff
@@ -2,9 +2,14 @@
 
 import os, sys, urllib, textwrap
 import codecs
-from six.moves.configparser import NoSectionError
 from os.path import join
+
+# BBB: Python 2 compatibility
```

Comment:
What does BBB mean?

Reply:
Grr, I wrote a response to this but somehow I lost it.

In the code bases I "grew up" in and several others since, I've seen "BBB" used roughly as "related to backwards compatibility".  Usually it means "remove this when we drop compatibility".  Sometimes it's used more generally as "to be removed when X is done".